spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Concat.scala

```diff
@@ -108,6 +110,27 @@ class Concat[T: ClassTag](val dimension: Int)(
     this.output
   }
 
+  private def checkOutputs(outputs : Array[Tensor[T]], dim : Int) : Unit = {
```

Comment:
Can we do this check at line 66? when update this.size also check other dimensinos

Reply:
@yiheng  makes sense,  we don't need to wait all module to compute before check, coming back shortly with refinement